app/scripts/controllers/mobileClients.js

```diff
@@ -73,6 +76,15 @@ angular.module('openshiftConsole')
                 }
                 ctrl.mobileClient = mobileClient;
               }));
+
+              watches.push(DataService.watch(serviceInstanceVersion, context, function (serviceInstances) {
+                $scope.serviceInstances = serviceInstances.by("metadata.name");
+                ctrl.mobileCIEnabled = _.some($scope.serviceInstances, function(serviceInstance) {
+                  console.log(serviceInstance.metadata.name + " :: ready = " + isServiceInstanceReady(serviceInstance));
+                  return /aerogear-digger/.test(serviceInstance.metadata.name) && isServiceInstanceReady(serviceInstance);
```

Comment:
@sedroche @JameelB Filtered now to avoid setting mobileCIEnabled before the service is ready.